sys/include/net/nanocoap.h

```diff
@@ -166,10 +166,13 @@ extern "C" {
  * @name    Nanocoap specific CoAP method flags used in coap_handlers array
  * @{
  */
-#define COAP_GET                (0x1)
-#define COAP_POST               (0x2)
-#define COAP_PUT                (0x4)
-#define COAP_DELETE             (0x8)
+#define COAP_GET                (0x01)
+#define COAP_POST               (0x02)
+#define COAP_PUT                (0x04)
+#define COAP_DELETE             (0x08)
+#define COAP_PATCH              (0x10)
```

Comment:
COAP_PATCH -> COAP_FETCH

Typo. I expect we will fix this one to stay consistent with the ordering of method codes.